packages/unity-react-core/src/components/GridLinks/GridLinks.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+
+import { GridLinks, GridLinksProps } from "./GridLinks";
+import {
+  gridLinksNumColumns,
+  gridLinksTextColor,
+  gridLinksBgColor,
+  gridLinksBgColorClassName,
+} from "./GridLinksConstants";
+
```

Comment:
See note in Constants file about removing `  gridLinksBgColor`, `gridLinksBgColorClassName`

Since this is not a prop on the component we need to manually set the controls

```
const gridLinksBgColorClassName = {
  None: "",
  "Gray 1": "bg-gray-1",
  "Gray 7": "bg-gray-7",
};

export default {
  title: "Components/Grid Links",
  component: GridLinks,
  argTypes: {
    bgColor: {
      name: "Background Color of the section",
      description: "Background color is not applied to the component, but to the section wrapper.",
      options: Object.keys(gridLinksBgColorClassName),
      mapping: gridLinksBgColorClassName,
      control: {
        type: "radio",
      },
    },
  },
};
```